src/components/Task.vue

```diff
@@ -56,12 +65,31 @@ export default defineComponent({
     NTag,
     TreeViewAlt
   },
+  methods:{
+    getAssigneeInitials(assignee: string) {
+      if (assignee) {
+        const nameSplit = assignee.split(" ");  
+        return nameSplit.length > 1 ? nameSplit[0][0] + nameSplit[nameSplit.length - 1][0] : nameSplit[0][0];
+      }
+    },
+    getAvatarColor(assignee: string) {
```

Comment:
I think you copied this from Asana? If so, let's put a clear comment stating that (otherwise there is lots of magic going on).

This also probably should live in the avatar-colors file you created